iModelCore/ECDb/ECDb/ClassViews.h

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See COPYRIGHT.md in the repository root for full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+#pragma once
+#include "ECDbInternalTypes.h"
+#include "ClassMap.h"
+#include <functional>
+
+BEGIN_BENTLEY_SQLITE_EC_NAMESPACE
+//======================================================================================
+// @bsiclass
+//======================================================================================
+struct ClassViews final {
+    constexpr static char ViewSchemaName[] = "ECDbMap";
```

Comment:
constexpr char[] seems inconsistent, @khanaffan can we change this code to Utf8CP instead?